### DISCUSSION TITLE
Can you set the start value of the auto increment tool?

### DISCUSSION THREAD

**Discussion by ghost** [flameshot-org/flameshot#3163] [Q&A]
I'm new to Flameshot and wonder how I ever survived with out it at this point.  Love it and use it all the time now.

My question is can you set the start value of the auto increment tool?  I take notes in a 'how to do this step by step' fashion and was wondering if there was a way to set the starting value when you use the auto increment tool.   So if I have steps 1-4 in the first screenshot, sometimes I'd like to continue the count by having steps 5-7 in the next screenshot.  Not a major issue if it doesn't exist, just wondering if that is built in and I'm missing it somehow.

Thanks for an awesome tool!!  I really do love it.

**Comment by Creteil** [comment]
+10000

**Comment by mmahmoudian** [comment]
> if I have steps 1-4 in the first screenshot, sometimes I'd like to continue the count by having steps 5-7 in the next screenshot.

There is no way to set the start, but the solution is super easy: click n times out of the region and then when it is the desired number place it where you need it. It is very fast, even faster than defining a new starting number.
